interfaces/acados_template/acados_template/builders.py

```diff
@@ -74,7 +75,7 @@ def get_cmd1_cmake(self):
     # Generate the command string for handling the build.
     def get_cmd2_build(self):
         import multiprocessing
-        cmd = f'cmake --build "{self._build_dir}" --config Release -j{multiprocessing.cpu_count()}'
```

Comment:
Did the `cpu_count` fail for you? The `import multiprocessing` is still there, so I guess that works.